.github/workflows/pr.yaml

```diff
@@ -76,11 +76,22 @@ jobs:
       - name: Deploy stack
         run: pnpm run deploy --stage ${{ env.STAGE }}
 
-      - name: Extract api endpoint
+      - name: Extract stack outputs
         id: sst-output
         run: |
           URL=$(jq -r '.["${{env.STAGE}}-birthday-slack-bot-MyStack"].ApiEndpoint' .sst/outputs.json)
           echo "apiEndpoint=$URL" >> "$GITHUB_OUTPUT"
+          MIGRATION_FUNCTION=$(jq -r '.["${{env.STAGE}}-birthday-slack-bot-MyStack"].MigrationFunctionName' .sst/outputs.json)
```

Comment:
This step is not needed, if you give Name prop to the new Function in mystack, then the sst wont prefix it with the stage, so the function will be just MigrateDb

Reply:
Y but then it won't be unique anymore. E.g for me it collided with @attilazsoltdanku -s version when tried to deploy earlier.

Reply:
Yup but it wasnt empty before